src/main/java/net/sargue/mailgun/content/MailContent.java

```diff
@@ -6,7 +6,9 @@
 /**
  * This class is deprecated. Don't use it. See below for instructions on how to
  * migrate current code.
- * <h3>Migration guide</h3>
+ *
```

Comment:
Why this change?

Reply:
When i run `./gradlew install` it fails with this error
![image](https://user-images.githubusercontent.com/63702645/113106266-3cf64480-920b-11eb-8bfc-d9670d40bf2a.png)

Not sure if that my local issue only or anything I can do about it.